README.md

```diff
@@ -136,20 +141,53 @@ It's mostly used for receiving events from the View, such as clicks.
 
 In the View:
 ```kotlin
-button.clicks() bindTo pm.buttonClicks.consumer
+button.clicks() bindTo pm.buttonClicks
 ```
 
 In the PresentationModel:
 ```kotlin
-val buttonClicks = Action<Unit>()
+val buttonClicks = action<Unit>()
 
+// Subscribe in onCreate
 buttonClicks.observable
     .subscribe {
         // handle click
     }
     .untilDestroy()
 ```
 
+#### Action initialisation block to avoid mistakes
+Typically, the Action triggers an asynchronous operations, such as a request to backend. In this case, the rx-chain will may throw an exception and app will crash. We can handle errors in `subscribe`, but this is not enough. After the first failure, the chain will be completed and stop processing clicks. Therefore, the correct handling involves the use of the `retry` operator and looks as follows:
```

Comment:
Typically, some Action triggers an asynchronous operation, such as a request to backend. In this case, the rx-chain may throw an exception and app will crash. It's possible to handle errors in the subscribe block, but this is not enough. After the first failure, the chain will be terminated and stop processing clicks. Therefore, the correct handling involves the use of the retry operator and looks as follows: